templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -843,6 +843,15 @@ export default {
         )
       }
 
+      if (this.node.popup) {
+        const { time } = this.node.popup
+        if (time == undefined) {
```

Comment:
```suggestion 
        if (time === "") {
```
`time` appears as an empty string rather than undefined.

Reply:
Good catch!